config/faf-clans/faf-clans.env

```diff
@@ -1,3 +1,7 @@
-WEBROOT=/usr/share/nginx/html
-TEMPLATE_NGINX_HTML=0
+API_BASE_URL=https://api.dev.faforever.com
+OAUTH_CLIENT_ID=83891c0c-feab-42e1-9ca7-515f94f808ef
+OAUTH_REDIRECT_URI=https://clans.faforever.com
```

Comment:
If you look at other .env files, you see that none of them references `faforever.com`. The reason behind this is that `faf-stack` should run locally and independently of any external server, without further configuration.

You will also see that none of them defines an actual OAuth client ID. So please follow this concept, and keep things clean and consistent.

fyi. @duk3luk3